jetty9-base/src/main/jetty-base/etc/gae-web.xml

```diff
@@ -2,6 +2,9 @@
 <!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
 <Configure class="org.eclipse.jetty.webapp.WebAppContext">
 
+  <!-- Throw Exception on Start -->
```

Comment:
This comment is a little confusing. Can we put something more descriptive like "Propagate application startup exceptions to server", or whatever makes sense.